app/views/wishlist_mailer/item_purchased_email.html.slim

```diff
@@ -0,0 +1,5 @@
+  h1 You have successfully purchased an item you reserved for a snowman.
+    p We have been notified that your purchase was successful. You will shortly be receiving details on when the item will be delivered to your secret snowman.
```

Comment:
you're nesting these paragraphs under the header, which means that *this entire email* will be formatted as a `h1`.